packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -167,6 +168,10 @@ export default class TSGenerator extends CodeGenerator {
   async compile(storage: Storage, opts: InstallerOptions = {}): Promise<void> {
     const installDir = storage.getIdentifierStorageDir();
 
+    await execa('npm', ['install', 'tsup', 'typescript', '-D'], {
```

Comment:
we need TS as a dev-dep so the `tsup` build runs properly:

![CleanShot 2023-10-16 at 15 09 46@2x](https://github.com/readmeio/api/assets/8854718/85ba5d53-1aff-4a63-8d19-0a8a908a24d3)

Reply:
we probably want to include this in that list of packages along with some CLI explainers as to why we need said packages, but this PR is just intended to get it working properly